ui/main/src/app/services/admin/SupervisedEntitiesService.ts

```diff
@@ -63,7 +74,12 @@ export class SupervisedEntitiesService {
 
                     return responseEntities.data;
                 } else {
-                    ErrorService.handleServerResponseError(responseEntities);
+                    logger.error(`Error while updating entity :  ${responseEntities.statusMessage}`);
```

Comment:
Maybe we should add in the log the id of the entity we're trying to update ?

Reply:
ok